lciafmt/iw.py

```diff
@@ -5,10 +5,16 @@
 import lciafmt.df as df
 import lciafmt.util as util
 
-print([x for x in pyodbc.drivers() if x.startswith('Microsoft Access Driver')])
+# Check for drivers and display help message if absent
+Driver_Check = ([x for x in pyodbc.drivers()])
+if any('Microsoft Access Driver' in word for word in Driver_Check):
+    print("Drivers Available")
+else:
+    print("Please install drivers to remotely connect to Access Database. Drivers only available on windows platform. "
+          "For instructions visit: https://github.com/mkleehammer/pyodbc/wiki/Connecting-to-Microsoft-Access")
```

Comment:
Can you put this section of code [8-14] inside the get function? Otherwise it will do this test upon importing iw.py as opposed to only when a user tries to get/read the impact world database.

Also let's change these print statements to log.debug and log.warn respectively (see line 33 for example)